src/client/utils/date/date.ts

```diff
@@ -59,3 +66,74 @@ export function formatTimeRange(timeRange: TimeRange, timezone: Timezone, displa
 
   return formatted;
 }
+
+// calendar utils
+
+export function monthToWeeks(startDate: Date, tzString: string): Date[][] {
+  const weeks: Date[][] = [];
+  var week: Date[] = [];
+  var firstDayOfMonth = wallTimePreciseToMonth(startDate, tzString);
+  for (var i = 1; i <= getCountDaysInMonth(firstDayOfMonth); i++) {
+    var activeDay = wallTimePreciseToDay(startDate, i, tzString);
+    // this assumes week starts on sunday..
```

Comment:
Maybe this should be linked to the locale ? Do we intent to support weeks that start on Mondays too ?